rst_files/generic_programming.rst

```diff
@@ -569,7 +569,7 @@ Hopefully this showcases the power of generic programming:  with a well-designed
 Reals and Algebraic Structures
 =======================================
  
-Thinking back to the mathematical motivation, a `field <https://en.wikipedia.org/wiki/field_\(mathematics\)>`_ is a ``ring`` with a few additional properties, among them
+Thinking back to the mathematical motivation, a `field <https://en.wikipedia.org/wiki/field_\(mathematics\>`_ is a ``ring`` with a few additional properties, among them
```

Comment:
Ditto

Reply:
@jlperla it doesn't work. I changed it. I will check it now.

Reply:
I don't think that the sphinx/jupinx has been fixed, so checking is premature.